packages/server/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
+    "esModuleInterop": true,
```

Comment:
can remove? this is in the tsconfig.base

Reply:
If I remove this, it doesn't work. It looks like the option in `tsconfig.base.json` is not in effect in the sub folder?🤯

Reply:
can you describe how it doesn't work?

Reply:
ahhh it's the stringifier.
bigger mystery is why the tsconfig isn't honoring the extends, but we can deal with that later